json-to-xml-converter/main.bal

```diff
@@ -0,0 +1,34 @@
+import ballerina/data.xmldata;
+import ballerina/http;
+import ballerina/io;
+import ballerina/log;
+
+listener http:Listener httpDefaultListener = http:getDefaultListener();
+
+service / on httpDefaultListener {
+    function init() returns error? {
```

Comment:
Let's remove this.

Reply:
Removed in https://github.com/wso2/ballerina-integrator-samples/pull/1/commits/aee9df823d5e8c58c0c03401206b222a8becf2b0